ponyo/generate_labeled_data.py

```diff
@@ -199,42 +196,49 @@ def simulate_compendium_labeled(
         new_centroid = np.zeros(latent_dim)
 
         for j in range(latent_dim):
-            new_centroid[j] = np.random.normal(
-                encoded_means[j], encoded_stds[j])
+            new_centroid[j] = np.random.normal(encoded_means[j], encoded_stds[j])
 
         shift_vec_df = new_centroid - centroid
 
         simulated_data_encoded_df = data_encoded_df.apply(
-            lambda x: x + shift_vec_df, axis=1)
+            lambda x: x + shift_vec_df, axis=1
+        )
 
         # Decode simulated data into raw gene space
         simulated_data_decoded = loaded_decode_model.predict_on_batch(
-            simulated_data_encoded_df)
+            simulated_data_encoded_df
+        )
 
-        simulated_data_decoded_df = pd.DataFrame(simulated_data_decoded,
-                                                 index=simulated_data_encoded_df.index,
-                                                 columns=selected_data_df.columns)
+        simulated_data_decoded_df = pd.DataFrame(
+            simulated_data_decoded,
+            index=simulated_data_encoded_df.index,
+            columns=selected_data_df.columns,
+        )
 
         # Add experiment label
-        simulated_data_decoded_df["experiment_id"] = selected_experiment_id + \
-            "_" + str(i)
+        simulated_data_decoded_df["experiment_id"] = (
+            selected_experiment_id + "_" + str(i)
+        )
```

Comment:
There's nothing wrong with this line of code, but you do a lot of string joining in these files so I figured I'd mention two other methods that might make your life easier.

One is to use string formatting, which keeps you from having to cast things with `str`.
For example `'{}_{}'.format(selected_experiment_id, i)` would do the same thing here.

If you're joining a lot of strings by a single character, you can also use `'_'.join(selected_experiment_id, str(i))` where the first character is the character to join things with and the parameters are a list of strings

Reply:
Thanks! I think i will leave the strings as is for now so that I can get this into pypi but I shall definitely keep these in mind for my future scripts!